package.json

```diff
@@ -18,23 +19,19 @@
   },
   "homepage": "https://github.com/datproject/dat-js#readme",
   "dependencies": {
-    "buffer-alloc-unsafe": "^1.0.0",
     "dat-encoding": "^5.0.1",
+    "discovery-swarm-web": "^1.0.4",
     "hypercore-crypto": "^1.0.0",
+    "hypercore-protocol": "^6.9.0",
     "hyperdrive": "^9.4.7",
-    "pump": "^1.0.3",
     "random-access-memory": "^2.4.0",
-    "run-parallel": "^1.1.9",
-    "signalhubws": "^1.0.10",
-    "sodium-universal": "^2.0.0",
-    "through2": "^2.0.0",
-    "webrtc-swarm": "^2.9.0",
-    "websocket-stream": "^5.1.2",
-    "xtend": "^4.0.1"
+    "random-access-web": "github:random-access-storage/random-access-web",
```

Comment:
Can we get this published before merging this PR?

Reply:
Whoops! I forgot to update the dependency